snafu/dns_perf_wrapper/trigger_dns_perf.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import sys
+import datetime
+import time
+import string
+import logging
+import json
+import subprocess
+from ttp import ttp
+
+logger = logging.getLogger("snafu")
+
+class Trigger_dns_perf():
+    # Initialize the arguments defined
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.cluster_name = args.cluster_name
+        self.server_address = args.server-address
```

Comment:
This works? Example of my concern 
```
>>> test-var=10
  File "<stdin>", line 1
    test-var=10
    ^
SyntaxError: cannot assign to operator
>>> test_var=10
>>> 

```

Reply:
Right, modified to avoid python interpreting hyphens as subtraction operator.